src/contexts/ItemsContext.jsx

```diff
@@ -0,0 +1,152 @@
+import React, { useContext } from "react";
+import PropTypes from "prop-types";
+
+// data in our store
+const items = [
+  {
+    id: "",
+    inventory: 100,
+    title: "Super Long Watch Name",
+    desc: "Series 6. Red",
+    longDesc:
+      "Available when you purchase any new iPhone, iPad, iPod Touch, Mac or Apple TV, 4.99/month after free trial.",
+    fullDesc: "",
```

Comment:
What the difference between longDesc and fullDesc?

Reply:
There is a full description button/link on the single item page.